dali/operators/decoder/image_decoder.cc

```diff
@@ -185,8 +185,8 @@ image format, it will decode the entire image and crop the selected ROI.
 
 .. note::
   ROI decoding is currently not compatible with hardware-based decoding. Using
-  :meth:`nvidia.dali.fn.image_decoder` automatically disables hardware accelerated
-  decoding. To use the hardware decoder, use the :meth:`nvidia.dali.fn.image_decoder` and
+  :meth:`nvidia.dali.fn.decoders.image` automatically disables hardware accelerated
```

Comment:
That's a typo (the doc before). It should be `image_decoder_crop`/`decoders.image_crop`

Reply:
done